models/multi_organ_segmentation/large_files.yml

```diff
@@ -0,0 +1,13 @@
+large_files:
+  - path: "models/search_code_18590.pt"
+    url: "https://drive.google.com/uc?export=download&id=1G2YZo1HKkvf5sMvaaRQeZp_YA9GDlvL-"
+    hash_val: "55dc7fe4bee4a93d25a1dd329dfdd159"
+    hash_type: "md5"
+  - path: "models/model.pt"
+    url: "https://drive.google.com/uc?export=download&id=1kH0yTyiXUNqdYXpnSXI2p5-vFwDYoCzl"
+    hash_val: "2a167107a34ac588130ec47509b21b9e"
```

Comment:
Hi @cshen0625 , could you please also help to check the weights and hash values?
The CI test is failed and it seems the hash value is wrong. I tried to download `model.pt` locally and checked `md5sum`, and I got: `e21d663b33d29a8ca4e28d60a14a3d66`

Reply:
Hi @yiheng-wang-nv , I'm sorry that I forgot to update the hash information. I have now updated the hash value.